### PR TITLE
fix(collector-version-form): fix autoUpgrade version initialization

### PR DESCRIPTION
**Pull request by sulmoJ**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [x] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [x] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
The purpose is to set autoUpgrade to true when creating a collector.

### CODE REVIEW

File: apps/web/src/services/asset-inventory/collector/shared/collector-forms/CollectorVersionForm.vue [hunk 1]
```diff
@@ -69,7 +69,7 @@ const initSelectedVersion = () => {
         const originVersion = collectorFormState.originCollector?.plugin_info?.version;
         collectorFormStore.$patch({
             version: originVersion ?? collectorFormState.versions[0] ?? '',
-            autoUpgrade: originAutoUpgrade ?? true,
+            autoUpgrade: props.getVersionsOnPluginIdChange ? true : (originAutoUpgrade ?? true),
```
Comment (WANZARGEN) [inline apps/web/src/services/asset-inventory/collector/shared/collector-forms/CollectorVersionForm.vue:72]: question: Why is this line required? 😮

**Review by WANZARGEN — COMMENTED** [review]
I left one question~!

File: apps/web/src/services/asset-inventory/collector/shared/collector-forms/CollectorVersionForm.vue (hunk 1 above)
Comment (sulmoJ) [reply apps/web/src/services/asset-inventory/collector/shared/collector-forms/CollectorVersionForm.vue:72]: This is because when you initialize createPage, autoUpgrade must be 'true'.

Comment (sulmoJ) [reply apps/web/src/services/asset-inventory/collector/shared/collector-forms/CollectorVersionForm.vue:72]: This is because the `autoUpgrade` initial state of the version form on the create page must be `true`

**Review by yuda110 — APPROVED** [review]
LGTM! I have same question with WZ😅

**Review by WANZARGEN — APPROVED** [review]
Much clearer!! 👍
